packages/docs/verite/patterns/travel-rule.md

```diff
@@ -2,53 +2,104 @@
+### Part 3: Exchange of Counterparty Credentials
 
-### Part 2: Exchange of Counterparty Credentials
+If the transaction does not meet the threshold required for the Travel Rule, then Part 3 is not necessary and does not apply, as Alice's FI can simply proceed to broadcasting the blockchain transaction. 
+
+If the Travel Rule does apply, however, both financial institutions are able to exchange counterparty verifiable credentials over callback URLs that allow for private off-chain secure communication. This can happen over any secure channel, and today is mostly done in consortium/network-like platforms that bundle the authentication of step 2 and the secure channel for step 3 (as well as data dictionaries or translation enginges for exchanging the actual information required by applicable reporting requirements, itself a massively complex value-add and rules engine).  This research could be called long-tail, as the rapid pace of regulatory change here means the existing protocols will probably be distracted updating their rules engines for some time and not have bandwidth to consider harmonizing on UX and consent flows!
+
+As a prototype of user-centric UX and an architecture that layers consent receipts on top of reporting artefacts, the Verite prototype models how identity-enabled and verifiable-credential-enabled wallets could surface meaningful and direct consent into such a flow. While traditional financial institutions typically require their users to consent to terms and conditions that grant the rights to transfer this kind of private information, this prototype of more proactive and per-interaction consent could pave the way to better support for non-custodial wallet compliance, or conformance to more aggressive regulation about meaningful consent.  To address this latter hypothetical requirement, the flow presents notifications as alerts to customers so that they have visibility into when their data is requested, and to provide them the choice to cancel sending or receiving transactions that require such counterparty credentials exchange.
 
 1. If the transaction's value triggers the Travel Rule threshold, Alice's FI responds to Bob's FI by composing a private message "M3" containing:
    - Transaction amount
    - Alice's originating counterparty verifiable credential
    - Presentation Request for beneficiary counterparty credentials
 2. Bob's FI receives M3 and verifies the originator counterparty verifiable credential, and then executes its internal compliance procedures.
-3. Once Bob's FI clears the information, it generates message "M4" containing Bob's beneficiary counterparty verifiable credential.
+3. Once Bob's FI clears the information, it generates message "M4" containing Bob's beneficiary counterparty verifiable credential (or other verifiable consent receipt).
```

Comment:
This is not needed for TravelRule. No need for the beneficiary VASP to send the beneficiary data back to the originator VASP. It does need to match if the beneficiary data provided by the originator VASP with the beneficiary info they have

Reply:
So Bob's FI only needs to send some sort of "APPROVE" / "REJECT" message back